test/test_mq.rs

```diff
@@ -112,7 +112,15 @@ fn test_mq_getattr() {
 // FIXME: Fix failures for mips in QEMU
 #[test]
 #[cfg_attr(
-    all(qemu, any(target_arch = "mips", target_arch = "mips64")),
+    all(
+        qemu,
+        any(
+            target_arch = "mips",
```

Comment:
This would be a good candidate for cfg_aliases, which we have discussed for other purposes.
https://docs.rs/cfg_aliases/latest/cfg_aliases/

Reply:
If we are gonna replace most big-chunk, common `cfg!`s with some aliases, then it might be better to do it in another, dedicated PR, where we can replace them all and get them well documented in `CONTRIBUTING.md`

Reply:
#2188 issue filed